cid/overlay/frameworks/base/core/res/res/values/config.xml

```diff
@@ -79,7 +79,7 @@
     <string translatable="false" name="config_radio_access_family">GSM | WCDMA | LTE | CDMA | EVDO</string>
```

Comment:
Title is wrong, it needs to say for which platform you resolve the build errors. Look at other commits formatting.